docker-compose.yml

```diff
@@ -153,6 +154,31 @@ services:
     volumes:
     - ./web/dkim/keys:/etc/opendkim/keys
 
+  telegraf:
+    image: telegraf
+    container_name: telegraf
+    restart: unless-stopped
+    environment:
+      HOST_ETC: /hostfs/etc
+      HOST_PROC: /hostfs/proc
+      HOST_SYS: /hostfs/sys
+      HOST_VAR: /hostfs/var
+      HOST_RUN: /hostfs/run
+      HOST_MOUNT_PREFIX: /hostfs
+      SERVER_NAME: ${SERVER_NAME:?SERVER_NAME unset}
+      TELEGRAF_NAME: ${TELEGRAF_NAME}
+      TELEGRAF_INFLUXDB_DB: ${TELEGRAF_INFLUXDB_DB:?TELEGRAF_INFLUXDB_DB unset}
```

Comment:
is there any way we can reduce the number of environment varaibles we need to add?

it would be nice if we could just do: `TELEGRAF=https://<user>:<pass><url>/<db_name>` or something

Reply:
not sure. could investigate but that could delay this further when we have other things to consider ie. kapacitor stuff.